interfaces/builtin/checkbox_support.go

```diff
@@ -46,15 +46,15 @@ dbus (send)
     path=/org/freedesktop/systemd1
     interface=org.freedesktop.systemd1.Manager
     member=StartTransientUnit
-    peer=(name=org.freedesktop.systemd1,label=unconfined),
```

Comment:
is that the issue that name is not matched properly?

Reply:
Yes, on 16.04 this just doesn't work. After 16.04 it seems not to matter. I had a quick look at dbus code but I have not finished yet. The dbus code has not changed for nearly a decade.

Reply:
This is now more debugged, but still not understood fully.